app/src/main/java/org/simple/clinic/storage/migrations/Migration_71.kt

```diff
@@ -0,0 +1,19 @@
+package org.simple.clinic.storage.migrations
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+import org.simple.clinic.storage.inTransaction
+import javax.inject.Inject
+
+@Suppress("Classname")
+class Migration_71 @Inject constructor() : Migration(70, 71) {
```

Comment:
Just FYI, since we squash and merge now, we can move the migration to a different commit for easier review.

Reply:
Will do that from next migrations 👍🏾